src/server/test/db/cikVaryTests.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const { mocha, expect, testDB } = require('../common');
+const { redoCikVary } = require('../../services/graph/redoCik');
+const Conversion = require('../../models/Conversion');
+const Unit = require('../../models/Unit');
+const ConversionSegment = require('../../models/ConversionSegment');
+const CikVary = require('../../models/CikVary');
+
+async function setupTestData(conn) {
```

Comment:
I worked out the test setup and what cik vary should be but it took a little time. Maybe a comment with the original and cik values would be valuable? Some simple version of how the design doc discussed its example. I can help if you want.